terraform/browser/add_response_headers.js

```diff
@@ -4,11 +4,6 @@ function handler(event) {
 
     // Set HTTP security headers
     // Since JavaScript doesn't allow for hyphens in variable names, we use the dict["key"] notation 
-    headers['strict-transport-security'] = { value: 'max-age=63072000; includeSubdomains; preload'}; 
```

Comment:
Is this line removal intentional?

Reply:
Yes, this response header is now created via the Terraform config

Reply:
Yes, I've moved this edit to a separate commit where it is replaced by the Terraform resource `aws_cloudfront_response_headers_policy`.